lib/miq_expression.rb

```diff
@@ -645,6 +649,7 @@ def field_from_virtual_reflection?(field)
   end
 
   def attribute_supported_by_sql?(field)
+    return false if col_details[field].nil?
```

Comment:
:+1: thanks. Guess I would have done the ternary in that case.
But this will do